scripts/ci/test_ref_doc.sh

```diff
@@ -14,12 +14,11 @@ pip install -e ./tools
 [ -d privates ] && pip install -qqq privates/*.whl
 pip install $ALL_MODULES
 
-pip install "sphinx==1.6.7" -q
+pip install sphinx==1.6.7 Jinja2==3.0.3
```

Comment:
We should never use `-q` (quite) in `pip install`, as this will hide the dependency details.